conanfile.py

```diff
@@ -3,7 +3,7 @@
 
 class PyBind11Conan(ConanFile):
     name = "pybind11"
-    upstream_version = "2.2.4"
+    upstream_version = "2.6.1"
```

Comment:
Not 100% how this works, but I suppose you should change the revision to 0?

Reply:
I always forgot if we start at 0 or 1

Reply:
I think we start at 0.